libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -609,6 +640,24 @@ bool AP_Mount::get_attitude_euler(uint8_t instance, float& roll_deg, float& pitc
     return true;
 }
 
+bool AP_Mount::get_target_attitude_euler(uint8_t instance, float& roll_deg, float& pitch_deg, float& yaw_deg)
+{
+    auto *backend = get_instance(instance);
+    if (backend == nullptr) {
+        return false;
+    }
+
+    Vector3f rpy;
+    if (backend->get_target_attitude_euler(rpy)) {
```

Comment:
```suggestion
    if (!backend->get_target_attitude_euler(rpy)) {
        return false;
    }
```

... or probably just `return backend->get_attitude_target_euler(rpy);` depending on how things go